src/infra/ChatAnalyticsReporter.ts

```diff
@@ -68,7 +67,8 @@ export class ChatAnalyticsReporter {
       this.endpoint,
       {
         ...event,
-        sessionId
+        sessionId,
+        authorization: `KEY ${this.apiKey}`
```

Comment:
Out of curiosity, why is it preferable to pass the key via request body vs in header?

Reply:
Beacon doesn't support auth headers which is why we needed to move this to be sent via the request body. Figured its simpler to just do it for both cases instead of conditionally for beacon.